std_msgs/String component_name
std_msgs/String component_type
mapping[] mappings
std_msgs/String[] whitelist
std_msgs/Bool is_start

---
std_msgs/Bool success
